concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1235,6 +1209,42 @@ jobs:
 
                   cf push
 
+      - task: deploy-healthcheck
+        config:
+          platform: linux
+          image: docker:///governmentpaas/cf-cli
+          inputs:
+            - name: paas-cf
+            - name: config
+            - name: bosh-CA
+          run:
+            path: sh
+            args:
+              - -e
+              - -u
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                . ./config/config.sh
+                echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                cf create-org testers
+                cf set-quota testers test_apps
+                cf create-space healthcheck -o testers
+                cf target -o testers -s healthcheck
+
+                cd paas-cf/tests/example-apps/healthcheck
+                cf push --no-start
+
+                cf create-service postgres M-dedicated-9.5 healthcheck-db
```

Comment:
Running the `cf create-service` and `cf bind-service` commands when the services and bindings already exists does not cause a problem. It prints a friendly message saying it does not need to do anything and then exits 0.